ui/ui-components/pages/import/[id]/edit.tsx

```diff
@@ -42,14 +35,13 @@ export default function Page(props) {
           <h2>Details</h2>
           <p>
             Creator:{" "}
-            <Link href={`/object/${_import.creatorId}`} prefetch={false}>
-              <a>
-                {_import.creatorType === "group"
-                  ? `"${groupName(_import.creatorId)}"`
-                  : null}{" "}
-                {_import.creatorId}
-              </a>
-            </Link>
+            {_import.creatorType === "run" ? (
+              <Link href="/run/[id]/edit" as={`/run/${_import.creatorId}/edit`}>
+                <a>{_import.creatorId}</a>
+              </Link>
+            ) : (
+              _import.creatorId
+            )}
```

Comment:
From what I could see, imports never have `creatorType: "group"` and the only `creatorTypes` we're using is `run`, `api` (in the `import:create` action, which I believe itself is unused) and now `task`. I changed this to what was already being used in `ui-components/components/import/list.tsx`.